### PR TITLE
Use katex in RangeEditor #3308

### PR DESCRIPTION
**Pull request by nikolas**
I've updated this component to use katex directly instead of better-react-mathjax. It appears to fix the issue in graph type 21 where the toggle broke the latex display in the gA3 value's label.

If this works well I'll adapt the other parts of the code to use this too.

### CODE REVIEW

**Comment by Evan-CTL** [comment]
Throwing errors in RangeEditor
Screenshot 2024-07-17 at 4 47 01 PM

**Comment by nikolas** [comment]
I saw this as well in development mode - compilation works in production mode with `npm run build`. I'll see if I can address this error.

**Comment by nikolas** [comment]
Alright, this branch should be working with `npm run dev` now. There are some settings being used in our dev webpack configuration that aren't present in production, and I've done some adjustments that fix this. We may be able to remove these settings, ultimately.

Just to make sure you get a completely new build and remove old build files, if you have any issues, do a `rm -rf media/js/build` before building. There was a `katex.mjs` file being added to this directory which interfered with the import.
